src/ui/common/src/components/workflows/operator/summaryList.tsx

```diff
@@ -19,20 +19,13 @@ type Props = {
   initiallyExpanded: boolean;
 };
 
-const listStyle = {
-  width: '100%',
-  maxWidth: 360,
-  bgcolor: 'background.paper',
-};
-
 const SummaryList: React.FC<Props> = ({
```

Comment:
How similar are the artifact and operator SummaryList components?

Perhaps we should combine the two components into one if they are similar enough.